cms/djangoapps/contentstore/tests/test_contentstore.py

```diff
@@ -1769,8 +1769,8 @@ def test_rerun_course_resets_advertised_date(self):
         source_course = CourseFactory.create(advertised_start="01-12-2015")
         destination_course_key = self.post_rerun_request(source_course.id)
         destination_course = self.store.get_course(destination_course_key)
-
-        self.assertEqual(None, destination_course.advertised_start)
+        # Advertised_start is String field so it will return empty string if its not set
+        self.assertEqual('', destination_course.advertised_start)
```

Comment:
Does this mean that `advertised_start` should be set to `''` everywhere that it's currently set to `None`? A recent change added this line:

https://github.com/edx/edx-platform/blob/master/cms/djangoapps/contentstore/views/course.py#L790